### PR TITLE
added tie-breaking rules

### PR DESCRIPTION
**Pull request by Ritikace01**
I have added the necessary tie-breaking rules in the leaderboard. Request you to review them and merge them by the earliest.

For you information, I have listed the tie-breaking rules for you, so that you can review them at your end.
There are basically three major ranking rules:-
1. Sort them according to their scores.
2. If scores are same, then rank on basis of who has solved issues from more levels.
3. If still not resolved, then rank on basis of PR count.
4. If still not resolved, then rank them alphabetically.

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -34,25 +41,64 @@ const UsersPRs = AllPRs.reduce((acc, pr) => {
...
+       else if(b.PRCount - a.PRCount < 0) return -1;
+       else if(b.PRCount - a.PRCount === 0) {
+         var a_name = a.Username.toLowerCase();
+         var b_name = b.Username.toLowerCase();
+         // compare on username - for now
+         if(a_name < b_name) return -1;
+         else if(a_name > b_name) return 1;
+         else return 0;
+       }
+     }
```
Comment (praveenscience) [inline App.js]: Great logic, but wrong place! 😅

**Review by praveenscience — CHANGES_REQUESTED** [review]
Let me show you where you should change it.
https://github.com/praveenscience/GSSoC21-Leaderboard-Projects/blob/gh-pages/leaderboard.js#L6-L9

File: App.js (hunk 1 above)
Comment (Vishal-raj-1) [reply App.js]: @Ritikace01 ??

**Comment by praveenscience** [comment]
Added to the generator manually. 😊

**Pull request closed without merging**